test/main_test.go

```diff
@@ -120,3 +120,87 @@ func getParsedFile(t testing.TB, filePath string) model.FileMetadatas {
 
 	return files
 }
+
+func getCombinedParser() *parser.Parser {
+	return parser.NewBuilder().
+		Add(&jsonParser.Parser{}).
+		Add(&yamlParser.Parser{}).
+		Add(terraformParser.NewDefault()).
+		Add(&dockerParser.Parser{}).
+		Build()
+}
+
+func getQueryContent(queryDir string) string {
+	fullQueryPath := filepath.Join(queryDir, query.QueryFileName)
+	return string(getFileContent(fullQueryPath))
+}
+
+func getSampleContent(queryDir string) []byte {
+	return getFileContent(queryDir)
+}
+
+func getFileContent(filePath string) []byte {
+	content, _ := ioutil.ReadFile(filePath)
+	return content
+}
+
+func sliceContains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func getPlatform(platform string) (string, error) {
+	var genericPath = "../assets/libraries/"
+	var content = "package generic.common"
+	var errorMessage error
+
+	if strings.Contains(platform, "commonQuery") {
+		mFile := filepath.FromSlash(genericPath + "common/library.rego")
+		content, err := ioutil.ReadFile(mFile)
+		if err != nil {
+			log.Err(err)
+		}
+		return string(content), err
+	} else if strings.Contains(platform, "ansible") {
+		mFile := filepath.FromSlash(genericPath + "ansible/library.rego")
+		content, err := ioutil.ReadFile(mFile)
+		if err != nil {
+			log.Err(err)
+		}
+		return string(content), err
+	} else if strings.Contains(platform, "cloudFormation") {
+		mFile := filepath.FromSlash(genericPath + "cloudformation/library.rego")
+		content, err := ioutil.ReadFile(mFile)
+		if err != nil {
+			log.Err(err)
+		}
+		return string(content), err
+	} else if strings.Contains(platform, "dockerfile") {
+		mFile := filepath.FromSlash(genericPath + "dockerfile/library.rego")
+		content, err := ioutil.ReadFile(mFile)
+		if err != nil {
+			log.Err(err)
+		}
+		return string(content), err
+	} else if strings.Contains(platform, "k8s") {
+		mFile := filepath.FromSlash(genericPath + "k8s/library.rego")
+		content, err := ioutil.ReadFile(mFile)
+		if err != nil {
+			log.Err(err)
+		}
+		return string(content), err
+	} else if strings.Contains(platform, "terraform") {
+		mFile := filepath.FromSlash(genericPath + "terraform/library.rego")
+		content, err := ioutil.ReadFile(mFile)
+		if err != nil {
+			log.Err(err)
+		}
+		return string(content), err
+	}
+
+	return content, errorMessage
+}
```

Comment:
I know this is not code added in this PR, but we could reduce it in the same way it was done in this [commit](https://github.com/Checkmarx/kics/pull/1779/commits/ff4a53f662c41b613b4d18b9fad49ae9ad579ecf).

Reply:
Sure thing